atb_school_product_and_details.xml

```diff
@@ -1602,5 +1602,718 @@
+              0000000 <!-- Week 31 - Summer vacation -->
+              0000000 <!-- Week 32 - Summer vacation -->
+              0000000 <!-- Week 33 - Summer vacation -->
+              1111100 <!-- Week 34 -->
+              1111100 <!-- Week 35 -->
+              1111100 <!-- Week 36 -->
+              1111100 <!-- Week 37 -->
+              1111100 <!-- Week 38 -->
+              1111100 <!-- Week 39 -->
+              1111100 <!-- Week 40 -->
+              1100000 <!-- Week 41 - Autumn (Høstferie) vacation: 11-13 Oct -->
+              1111100 <!-- Week 42 -->
+              1111100 <!-- Week 43 -->
+              1111100 <!-- Week 44 -->
+              1111100 <!-- Week 45 -->
+              1111000 <!-- Week 46 - Elevfri: 17th Nov -->
+              1111100 <!-- Week 47 -->
+              1111100 <!-- Week 48 -->
+              1111100 <!-- Week 49 -->
+              1111100 <!-- Week 50 -->
+              1110000 <!-- Week 51 - Siste dag før jul 20.desember -->
+              0000000 <!-- Week 52 - Christmas vacation -->
+              0011100 <!-- Week 1 - Christmas vacation - Skolestart 3.jan. -->
+              1111100 <!-- Week 2 -->
+              1111100 <!-- Week 3 -->
+              1111100 <!-- Week 4 -->
+              1111100 <!-- Week 5 -->
+              1111100 <!-- Week 6 -->
+              1111100 <!-- Week 7 -->
+              1111100 <!-- Week 8 -->
+              0000000 <!-- Week 9 - Winter vacation: 26 Feb - 1 Mar -->
+              1111100 <!-- Week 10 -->
+              1111100 <!-- Week 11 -->
+              1111100 <!-- Week 12 -->
+              0000000 <!-- Week 13 - Easter -->
+              0011100 <!-- Week 14 - Easter -->
+              1111100 <!-- Week 15 -->
+              1111100 <!-- Week 16 -->
+              1111100 <!-- Week 17 -->
+              1101100 <!-- Week 18 - Off. høgtidsdag: 1 May -->
+              1110000 <!-- Week 19 - Ascension: 9 May, Elevfri: 10 May -->
+              1111000 <!-- Week 20 - Independence day: 17 May -->
+              0111100 <!-- Week 21 - 2. pinsedag: 20 May -->
+              1111100 <!-- Week 22 -->
+              1111100 <!-- Week 23 -->
+              1111100 <!-- Week 24 -->
+              1111100 <!-- Week 25 -->
+              0000000 <!-- Week 26 - Summer vacation starting -->
+            </ValidDayBits>
+          </UicOperatingPeriod>
+        </operatingPeriods>
+        <dayTypeAssignments>
+          <DayTypeAssignment version="1" id="NFK:DayTypeAssignment:SchoolDayNFK2023-2024" order="1">
+            <OperatingPeriodRef ref="NFK:UicOperatingPeriod:SchoolDayNFK2023-2024"/>
+            <DayTypeRef version="1" ref="NFK:FareDayType:SchoolDayNFK2023-2024"/>
+          </DayTypeAssignment>
+        </dayTypeAssignments>
+      </ServiceCalendar>
+    </ServiceCalendarFrame>
+
+    <!-- Nordland kommune: Lurøy, Sømna 2023-2024 -->
+    <ServiceCalendarFrame version="1" id="NFK:ServiceCalendarFrame:SchoolDayNFKLuroySomna2023-2024">
```

Comment:
Guess we can skip having NFK in all of the calendar names, apart from the common calendar?

Reply:
Good point. Changed.